External Code/SCPinions/SCPacketUtility/SCTracerouteUtility.m

```diff
@@ -85,6 +84,11 @@ -(void)stop{
     self.packetUtility = nil;
 }
 
+-(void)forcedTimeout {
+    if ( (self.delegate != nil) && [self.delegate respondsToSelector:@selector(tracerouteDidTimeout:)]) {
```

Comment:
I don't believe you need to check for nil since sending messages to nil in Objective-C is a no-op?